sentry_chassis_controller:
  type: sentry_chassis_controller/SentryChassisController 

  # 底盘参数
  wheel_track: 0.362
  wheel_base: 0.362
  wheel_radius: 0.055
  # 坐标系选择: global->全局坐标系, local->底盘坐标系
  coordinate_system: global
  # PID 参数
  pid:
    front_left_wheel:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
    front_right_wheel:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
    back_left_wheel:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
    back_right_wheel:
      p: 10.0    
      i: 0.5 
      d: 0.1
      i_max: 2.0
      i_min: -2.0
    front_left_pivot:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
    front_right_pivot:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
    back_left_pivot:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
    back_right_pivot:
      p: 1.0    
      i: 0.0
      d: 0.0
      i_max: 2.0
      i_min: -2.0
      
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50            